pkg/config/env.go

```diff
@@ -212,4 +212,22 @@ var Config = struct {
 	// UI path  => localhost:18000/foo"
 	// API path => localhost:18000/foo/api/v1"
 	WebPrefix string `env:"FLAGR_WEB_PREFIX" envDefault:""`
+
+	// rules for validating flags on enable - when a flag is updated or enabled
+	// (putFlag or setFlagEnabled), the flag is evaluated against these
+	// optional rules. The operator indicates if all the rules must pass ('AND' value)
+	// or if at least one rule must pass ('OR' value)
+	//
+	// Example rule:
+	// FLAGR_ENABLED_FLAG_VALIDATION_RULES='any("regexMatch(Value, \"^JIRA:[a-zA-Z]{3}[a-zA-Z]*$\")", Tags)'
```

Comment:
if you already have `any` or `all`, is the `OR`/`AND` operator still needed?

Reply:
In the spike we reviewed to decide on this approach, the team thought that having multiple rules + an `OR/AND` would make writing any individual rule less complicated.  Of course, since govaluate supports those operations you could just stuff the entire thing into the evaluation expression.  I'd be inclined to go that route (take out OR/AND operator) since it takes out a layer of complexity (you have to understand only govaluate).